portfolio/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.servlets;
+
+public final class Constants {
+  private Constants() {}
+  public static final String stats = "/WEB-INF/cases.csv";
```

Comment:
constant names should use all uppercase letters. See https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names